server/lib/src/migration_data/dl8/schema.rs

```diff
@@ -0,0 +1,1506 @@
+//! Schema Entries
+use crate::constants::entries::{Attribute, EntryClass};
+use crate::constants::uuids::*;
+use crate::schema::{SchemaAttribute, SchemaClass};
+use crate::value::IndexType;
+use crate::value::SyntaxType;
+
+lazy_static!(
+
+pub static ref SCHEMA_ATTR_DISPLAYNAME: SchemaAttribute = SchemaAttribute {
+    uuid: UUID_SCHEMA_ATTR_DISPLAYNAME,
+    name: Attribute::DisplayName,
+    description: "The publicly visible display name of this person".to_string(),
+
+    index: vec![IndexType::Equality],
+    sync_allowed: true,
+    syntax: SyntaxType::Utf8String,
+    ..Default::default()
+};
+
+pub static ref SCHEMA_ATTR_DISPLAYNAME_DL7: SchemaAttribute = SchemaAttribute {
```

Comment:
`DL8` if anything. But now that they have been moved into seperate modules we shouldn't need the naming right? would be `dl8::SCHEMA_ATTR_DISPLAYNAME`

This also goes for the other Domain Levels

Reply:
yes, but I just didn't prune all these. They'll age out naturally in time :)

Reply:
They'll age like milk. Just want to make sure that we don't set a bad precident for the next set of schema stuff :P

Reply:
I already cleaned up dl10 is more the point.